lib/Client.php

```diff
@@ -49,7 +49,7 @@ public function getDefaultTimeout()
 
     /**
      * @param RequestInterface $apiRequest
-     * @return \stdClass
+     * @return mixed
```

Comment:
The `send method` returns the result of an `json_decode` (https://github.com/williamokano/pagarme-php/blob/14ccc274762538c5176178d74214d660626cb138/lib/Client.php#L65).

According to the php documentation for the `json_decode` function, the return is `mixed`.

Reference: [http://php.net/json_decode](http://php.net/json_decode)

But, as you pointed, other methods can return their own type. If needed, I'll change it back to `\stdClass`